src/app/teams/teams.service.ts

```diff
@@ -27,6 +26,19 @@ export class TeamsService {
         displayName: displayName,
         email: email
       });
+
+      const leadMembersCollection: AngularFirestoreCollection<Membership> = this.afs.collection('membership', ref =>
+        ref.where('teamId', '==', teamId).where('isLead', '==', true));
+
+      const leadsObservable = leadMembersCollection.snapshotChanges().pipe(
+        map((members) => members.map(item => item.payload.doc.data()))
+      );
+
+      leadsObservable.subscribe(leads => {
+        leads.forEach(lead => {
```

Comment:
please pull in latest updates on email service where you can pass in an array  of recipients